hack/test/e2e.sh

```diff
@@ -225,26 +225,20 @@ function run_extensions_test {
   curl http://172.20.1.2/ | grep Hello
 }
 
-function run_day_two_tests {
-  rm -rf "${TMP}/day-two-state"
-  # pulumi needs to be in $PATH temporarily
-  PATH="${PATH}:$(dirname ${PULUMI})" ${D2CTL} up --state-path "${TMP}/day-two-state" --config-path "${PWD}/hack/test/day-two/config.yaml"
-
-  ${KUBECTL} --namespace loki wait --timeout 900s --for=jsonpath='{.status.availableReplicas}=1' deployment/loki-kube-state-metrics
-  ${KUBECTL} --namespace loki wait --timeout 900s --for=jsonpath='{.status.availableReplicas}=1' deployment/loki-prometheus-server
-  ${KUBECTL} --namespace loki wait --timeout 900s --for=jsonpath='{.status.availableReplicas}=1' deployment/loki-grafana
-
-  ${KUBECTL} --namespace metallb wait --timeout 900s --for=jsonpath='{.status.availableReplicas}=1' deployment/metallb-controller
-
-  ${KUBECTL} --namespace ingress wait --timeout 900s --for=jsonpath='{.status.availableReplicas}=1' deployment/ingress-nginx-controller
-
+function run_csi_tests {
+  rm -rf "${TMP}/rook"
+  git clone --depth=1 --single-branch --branch v1.8.2 https://github.com/rook/rook.git "${TMP}/rook"
```

Comment:
seems 1.8.2 was the one that mostly worked, probably helm installs some different version

Reply:
Mostly due to the addition of the default resource specs in `1.9.0`.